machines/VCItemMachine/ExistingMosipVCItem/ExistingMosipVCItemMachine.ts

```diff
@@ -1381,6 +1404,11 @@ export const ExistingMosipVCItemMachine =
                 case 'printing':
                   callback(model.events.DOWNLOAD_READY());
                   break;
+                case 'FAILED':
```

Comment:
Since FAILED and default case are having same implementation 

> case 'FAILED':
>default:
>    callback(model.events.FAILED()); 

we can mention it this way.